spec/features/idv/doc_capture/document_capture_step_spec.rb

```diff
@@ -131,7 +131,6 @@
 
     it 'proceeds to the next page with valid info and logs analytics info' do
       allow(Analytics).to receive(:new).and_return(fake_analytics)
-      expect(Analytics).to receive(:new).with(hash_including(user: user))
```

Comment:
Can we otherwise ensure in this spec that the analytics hash includes the user?

From what I can tell, it's not currently. I think it's because the controller's FlowStateMachine mixins are assuming to have been able to make use of the previous `analytics_user`.

Is there another way we could make sure those analytics tracking is using the correct user? Or should we leave that one as it was with `analytics_user`?

Reply:
It looks like analytics was added to fsm with no user_id override.  I'll fix it.